### DISCUSSION TITLE
Authentication - Windows Hello / 1Password integration

### DISCUSSION THREAD

**Discussion by rswio** [mRemoteNG/mRemoteNG#2504] [Ideas]
I love this RDP client, but for me, password are an issue. I don't like to save them in the client, so would find it useful if you could force Windows Hello for using stored credentials or perhaps even if you could integrate with 1Password for the credentials.

**Comment by Kvarkas** [comment]
hi, working on such exatly at the moment ;)

**Reply by rswio** [reply]
Hey, I was wondering how you were getting on with this and if the new version is something that you are still working on?

**Reply by Kvarkas** [reply]
at this moment implemnting fundamental changes into structure of the code and how mR handling a sensitive information. Keep trying to release this year.
